docs/Design/Design-Guide.md

```diff
@@ -1,151 +1,151 @@
-# Design guide
```

Comment:
This change actually doesn't change anything, but is included because the PR went into raft a little differently. Merging this will result in the same document, but will sync the repos

Reply:
lgtm 👍🏾